plugins/pulp_deb/plugins/distributors/steps.py

```
@@ -58,13 +60,25 @@ def process_main(self):
         Publish all the deb metadata or create a blank deb if this has never been synced
         """
         # Write out repo metadata into the working directory
+        packfile = os.path.join(self.get_working_dir(), "Packages")
+        dpkg_out = open(packfile, 'w')
+        proc = subprocess.Popen(['dpkg-scanpackages', '-m', '.'], cwd=self.get_working_dir(),
+                                stdout=subprocess.PIPE)
+        (out, err) = proc.communicate()
```

out and err are never checked